demo/site/src/components/common/CookiePlaceholders.tsx

```diff
@@ -0,0 +1,32 @@
+import { useCookieApi } from "@comet/cms-site";
+import styled from "styled-components";
+
+type Props = {
+    variant: "loading" | "fallback";
```

Comment:
I would rather create two independent components for those variants, as they have nothing in common (execpt the Root element styling, which can be shared)

Reply:
Done in 7653a35834a6048f67858bbd7f9c6c1118a71155